lib/MiLight/MiLightRadioInterface.h

```diff
@@ -0,0 +1,29 @@
+
+#ifdef ARDUINO
+#include "Arduino.h"
+#else
+#include <stdint.h>
+#include <stdlib.h>
+#endif
+
+#ifndef MILIGHTRADIOINTERFACE_H_
+#define MILIGHTRADIOINTERFACE_H_
+
+class MiLightRadioInterface
```

Comment:
What do you think about the name `RadioProvider`? I'm finding it a little confusing that there's a `MiLightRadioInterface` with an impl named `MiLightRadio` (maybe just a C++ pattern that I'm not familiar with?)